tex/alg-NT/frobenius.tex

```diff
@@ -599,8 +599,18 @@ \section{Frobenius elements control factorization}
 Once again we have a partition of $n$;
 we call this the \vocab{factorization pattern} of $f$ modulo $p$.
 So, to see the factorization pattern of an unramified $p$ in $\OO_E$,
-we just have to know the factorization pattern of the $f \pmod p$.
+we just have to know the factorization pattern of the $f \pmod p$.\\
+To prove our theorem, we will show that the factorization pattern of $f \pmod p$ corresponds exactly to the cycle decomposition of the action of $\Frob_{\kP}$ on the roots of $f$ and that the roots of $f$ correspond exactly to the cosets of $H$ in $G$.\\ 
+To do this, suppose $S = \{ \alpha_1,\alpha_2,\cdots, \alpha_n\}$ are the roots of $f$ (distinct roots since $f$ is irreducible over $\QQ$). We let $\Frob_{\kP}$ act on $S$. This splits $S$ into orbits $S_1,S_2,\cdots,S_k$. Construct polynomials $f_i$ with coefficients in $E$ having roots exactly the elements of $S_i$. This forms a factorization of $f$ over $E$. \[
+f = f_1f_2\cdots f_k
```

Comment:
This seems to be a sentence fragment, can you make this into a complete sentence with the preceding line?

Reply:
Maybe something like, `This forms a factorization of $f$ over $E$, say \[ f = f_1 f_2 \dots f_k. \]`

Also in general it's better to use `\dots` rather than explicit `\cdots` or `\ldots` unless the auto-detection is wrong.

Alos, replace `$S_1,S_2,\cdots,S_k` with `$S_1$, $S_2$, \dots, $S_k$` so the comma spacing is rendered correctly.